homeassistant/components/somfy/cover.py

```diff
@@ -76,6 +87,8 @@ def is_closed(self):
         is_closed = None
         if self.has_capability("position"):
             is_closed = self.cover.is_closed()
+        elif self.shadow:
+            is_closed = self._closed
```

Comment:
I think there is no consensus on this, but on some platforms (eg: [RFLink](https://github.com/home-assistant/home-assistant/blob/472fe7a0fa91a7a7da8d388ef0c031ac29f4d524/homeassistant/components/rflink/__init__.py#L487)) any position that is not CLOSED is considered OPEN. Following this criterion, we can consider that whenever a STOP command is sent, the cover will be OPEN since it will remain in an intermediate state nor CLOSED or OPEN.

Reply:
I think we are in agreement, that is what my proposed change does.